src/test/regress/expected/multi_complex_expressions.out

```diff
@@ -347,3 +347,84 @@ SELECT count(*) FROM lineitem JOIN orders ON l_orderkey = o_orderkey
 SELECT count(*) FROM lineitem, orders WHERE l_orderkey + 1 = o_orderkey;
 ERROR:  cannot perform local joins that involve expressions
 DETAIL:  local joins can be performed between columns only
+-- Check that we can issue limit/offset queries
+-- SIMPLE OFFSET without LIMIT and ORDER BY
+SELECT o_orderkey FROM orders WHERE o_orderkey > 9000 AND o_orderkey < 9100 OFFSET 10;
+ o_orderkey 
+------------
+       9024
+       9025
+       9026
+       9027
+       9028
+       9029
+       9030
+       9031
+       9056
+       9057
+(10 rows)
+
+-- SIMPLE LIMIT/OFFSET with ORDER BY
+SELECT o_orderkey FROM orders ORDER BY o_orderkey LIMIT 10 OFFSET 20;
+ o_orderkey 
+------------
+         69
+         70
+         71
+         96
+         97
+         98
+         99
+        100
+        101
+        102
+(10 rows)
+
+-- LIMIT/OFFSET with Subqueries
+SELECT 
```

Comment:
Do we support subqueries, or does this work because the subquery is flattened out?